docs/griptape-framework/structures/image-generation-drivers.md

```diff
@@ -0,0 +1,178 @@
+## Overview
+
+Image generation drivers are used by [image generation engines](../data/image-generation-engines.md) to build and execute API calls to image generation models.
+
+Use a driver to build an engine, then pass it to a tool for use by an [Agent](../structures/agents.md):
+
+```python
+from griptape.structures import Agent
+from griptape.engines import PromptImageGenerationEngine
+from griptape.drivers import OpenAiDalleImageGenerationDriver
+from griptape.tools import PromptImageGenerationClient, FileManager
+
+driver = OpenAiDalleImageGenerationDriver(
+    model="dall-e-3",
+)
+
+engine = PromptImageGenerationEngine(image_generation_driver=driver)
+
+agent = Agent(tools=[
+    PromptImageGenerationClient(image_generation_engine=engine),
+    FileManager(),
+])
+
+agent.run("Generate a watercolor painting of a dog riding a skateboard. Save the image as rad-dog.png.")
+```
+
+### Amazon Bedrock
+
+The Amazon Bedrock image generation driver provides multi-model access to image generation models hosted by Amazon Bedrock. This driver manages the API calls to the Bedrock API, while the specific model drivers below format the API requests and parse the responses.
+
+#### Bedrock Stable Diffusion Model Driver
+
+The Bedrock Stable Diffusion model driver provides support for Stable Diffusion models hosted by Amazon Bedrock. This model driver supports configurations specific to Stable Diffusion, like style presets, clip guidance presets, sampler, and more.
+
+This model driver supports negative prompts. When provided (for example, when used with an [image generation engine](../data/image-generation-engines.md) configured with negative rulesets), the image generation request will include negatively-weighted prompts describing features or characteristics to avoid in the resulting generation.
+
```

Comment:
do we want to illustrate the negative prompts in action? Perhaps one run without, one with?

Reply:
Added an example including negative rules